src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -29,6 +29,13 @@ public class ConfigurationLiveTests
             }
         };
 
+        private static string GetEnvironment()
```

Comment:
Are there any other tests using `AZ_CONFIG_CONNECTION` env variable?

Reply:
Yes. The [watcher](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationWatcherTests.cs) tests and the [sample ](https://github.com/Azure/azure-sdk-for-net/tree/master/src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/samples)tests

Reply:
I think we should change all of them to use the same method of getting connection string. 

Currently, some of them throw ArgumentNullException when environment variable is not set:

```
System.ArgumentNullException : Value cannot be null.
Parameter name: connectionString
   at Azure.ApplicationModel.Configuration.ConfigurationClient..ctor(String connectionString, ConfigurationPipelineOptions options) in D:\github\azure\azure-sdk-for-net\src\SDKs\Azure.ApplicationModel.Configuration\data-plane\Azure.ApplicationModel.Configuration\ConfigurationClient.cs:line 32
   at Azure.ApplicationModel.Configuration.Samples.ConfigurationSamples.<ConfiguringPipeline>d__5.MoveNext() in D:\github\azure\azure-sdk-for-net\src\SDKs\Azure.ApplicationModel.Configuration\data-plane\Azure.ApplicationModel.Configuration.Tests\samples\Sample7_ConfiguringPipeline.cs:line 49
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at NUnit.Framework.Internal.AsyncInvocationRegion.AsyncTaskInvocationRegion.WaitForPendingOperationsToComplete(Object invocationResult)
   at NUnit.Framework.Internal.Commands.TestMethodCommand.RunAsyncTestMethod(TestExecutionContext context)

```

Reply:
Sure thing. I'll do that in my next PR with other refactoring to the tests. Issue https://github.com/Azure/azure-sdk-for-net/issues/5665